src/main/assembly/dist/bin/easy-pid-generator

```diff
@@ -1,21 +1,21 @@
 #!/usr/bin/env bash
 
 BINPATH=`readlink -f $0`
-APPHOME=`dirname \`dirname $BINPATH \``
-MODULE=`basename $APPHOME`
+APPHOME=`dirname \`dirname ${BINPATH} \``
```

Comment:
IntelliJ keeps telling me to do that.
However, now that I look into it some more, I'm not sure about it. IntelliJ tells me to change `$DATADIR` to `${DATADIR}`, but the image below says the opposite... Maybe undo this change and disable this check in IntelliJ?

See also: https://github.com/jansorg/BashSupport/issues/86

![image](https://user-images.githubusercontent.com/5938204/29419268-f84693b6-836e-11e7-9a19-a9637525696f.png)